lizmap/modules/view/classes/lizmapMapDockItem.class.php

```diff
@@ -17,10 +17,11 @@ class lizmapMapDockItem
     public $order = 0;
     public $css = '';
     public $js = '';
+    public $jsParams = array();
     public $icon = '';
```

Comment:
@mdouchin `$icon` and `$menuIconClasses` are not in the constructor. It seems they have been forgotten. Should we add them or remove them as they seem unused till now ?